src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -380,31 +420,57 @@ RoverPositionControl::control_attitude(const vehicle_attitude_s &att, const vehi
 	// quaternion attitude control law, qe is rotation from q to qd
 	const Quatf qe = Quatf(att.q).inversed() * Quatf(att_sp.q_d);
 	const Eulerf euler_sp = qe;
+	// PX4_INFO("Yaw error: %f", double(euler_sp(2)));
+
+	_rates_sp.roll = 0.0;
+	_rates_sp.pitch = 0.0;
+	_rates_sp.yaw = euler_sp(2) / _param_max_turn_angle.get();
```

Comment:
```suggestion
	_rates_sp.yaw = euler_sp(2) * _param_att_p.get();
```
In the original logic, this `angle-error / max-turn-angle` was set as the actuator output. But when we want to command rate setpoint, this should rather be `attitude P-gain * angle-error`.